yt/frontends/open_pmd/data_structures.py

```diff
@@ -250,7 +250,7 @@ def _count_grids(self):
         self.vpg = int(self.dataset.gridsize / 4)  # 4Byte per value (f32)
 
         # Meshes of the same size do not need separate chunks
-        for (shape, spacing, offset, unit_si) in set(self.meshshapes.values()):
+        for (shape, _spacing, _offset, _unit_si) in set(self.meshshapes.values()):
```

Comment:
```suggestion
        for shape, *_ in set(self.meshshapes.values()):
```

Reply:
Also the code doesn't seem to be in agreement with the comment, shouldn't this be instead
```suggestion
        shapes = set(shape for (shape, *_) in self.meshshapes.values())
        for shape in shapes:
```